_docs/templates/_writers_guide.md.jinja

```diff
@@ -0,0 +1,149 @@
+(writers-contribution-guide)=
+## Writers
+
+Writer plugins add support for exporting data from napari.
+They are invoked whenever `viewer.layers.save('some/path.ext')`
+is used on the command line, or when a user requests to save one
+or more layers in the graphical user interface with
+`File -> Save Selected Layer(s)...` or `Save All Layers...`
+
+```{important}
+This guide describes the second generation (`npe2`) plugin specification.
+New plugins should no longer use the old `napari_get_writer` hook
+specification from the first generation `napari_plugin_engine`.
+```
+
+### Writer plugin function signatures
+
+Writer plugins are *functions* that accept a destination path and
+data with attributes from one or more layers in the viewer, write that data
+to the path, and -- if successful -- return a list of strings containing
+the path(s) that were successfully written.  They must follow one of two
```

Comment:
this sentence is pretty complicated. A numbered list might be good.

Reply:
done than ks